src/Node/EventNode.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * Date: 4/7/20
+ * Time: 5:41 PM
+ */
+
+namespace TwigBridge\Node;
+
+use Twig\Compiler;
+use Twig\Node\Node;
+use Illuminate\View\View;
+
+class EventNode extends Node
+{
+    public function compile(Compiler $compiler)
+    {
+        $compiler->write("if (!isset(\$context['__env']) && method_exists(\$this->env, 'mergeShared')) {\n")
+            ->indent()
+            ->write("\$context = \$this->env->mergeShared(\$context);\n")
```

Comment:
Could $context may conflict with a variable appended by another Node? I will check compiled template code to make sure...